doc/source/admin/python.md

```diff
@@ -0,0 +1,54 @@
+# Supported Python versions
+
+Galaxy's core functionality is currently supported on Python **2.7** (although
+deprecated) and **>=3.5** .
+
+If Galaxy complains about the version of Python you are using:
+
+1. Completely remove the Python virtualenv used by Galaxy (which can be
+   configured with the `GALAXY_VIRTUAL_ENV` environment variable and defaults to
+   `.venv`), e.g. with: `rm -rf /path/to/galaxy/.venv`
+
+2. If you were using a Python from a conda environment (which can be configured
+   with the `GALAXY_CONDA_ENV` environment variable and defaults to `_galaxy_`),
+   remove it, e.g. with: `conda env remove -n _galaxy_`
+
+3. Let Galaxy know which Python to use in one of the following methods:
+
+    - If you want to use Python from conda, just activate the `base` environment
+      and Galaxy will create a new conda environment for itself.
+    - Otherwise:
+        1. Make sure a supported version of Python is installed.
+        2. Verify that the Python interpreter you want to use is first in the
+           output of `which -a python3 python` (or `which -a python` if you are
+           using Galaxy <=20.01). If this is not the case:
```

Comment:
It's a bit silly to have the instructions for >=20.05 first, but I guess otherwise it becomes a mess to merge forward.

Reply:
Since I cherry-picked 765f59175c3b3bdb1d83a1d1cdf824f3d522b96d we can change the versions now